docs/release-notes/8.0.36-28.md

```diff
@@ -0,0 +1,62 @@
+# Percona Server for MySQL 8.0.36-28 (2024-02-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.36-28](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.36 [Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-36.html) and enterprise-grade features developed by Percona.
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.36 and included in Percona Server for MySQL are the following:'
+
+* OpenSSL is now version 3.0.12.
```

Comment:
The issue is that we almost always use the OpenSSL libraries from the OS.
Even Oracle says it more carefully
```
For platforms on which OpenSSL libraries are bundled, the linked OpenSSL library for MySQL Server has been updated to version 3.0.12
```
So, I would simply remove this item from the highlights.